### PR TITLE
Fix race condition on ProxySetup creation

### PR DESCRIPTION
**Pull request by unautre**
Related to [issue 2550](https://github.com/bndtools/bnd/issues/2550).

### CODE REVIEW

**Comment by bjhargrave** [comment]
You need to rebase and remove the merge commit.

You also need to amend your commits so that each commit message includes a `Signed-off-by` as required by CONTRIBUTING. Then push force your updated commits. Thanks.

File: biz.aQute.bndlib/src/aQute/bnd/connection/settings/ConnectionSettings.java [hunk 1]
```diff
@@ -354,11 +354,11 @@ public ProxySetup forURL(URL url) throws Exception {
 				return null;
 			}
 
-			// not synchronized because conflicts only do some double work
+			// not synchronized because conflicts only do some double work <- now it's true.
```
Comment (bjhargrave) [inline biz.aQute.bndlib/src/aQute/bnd/connection/settings/ConnectionSettings.java]: This comment wont mean much to someone reading this in the future :-)

**Comment by unautre** [comment]
Hello,
After learning git a bit more, I removed the merge commit and added the Signed-off-by.

**Comment by bjhargrave** [comment]
Thanks for the PR!

**Pull request merged**